recipes/base_install.rb

```diff
@@ -18,8 +18,6 @@
 include_recipe "aws-parallelcluster::setup_envars"
 include_recipe "aws-parallelcluster::sudoers_install"
 
-return if node['conditions']['ami_bootstrapped']
```

Comment:
I think this is still used when build-image process is executed using a pcluster backed AMI as source AMI: the install recipes won't be executed again.
We can debate if this is the right place where to put this check or better move it upstream in a common entrypoint